backend/compact-connect/stacks/persistent_stack/board_users.py

```diff
@@ -42,7 +45,9 @@ def __init__(
         self._add_scope_customization()
         # Do not allow resource server scopes via the client - they are assigned via token customization
         # to allow for user attribute-based access
-        self.ui_client = self.add_ui_client()
+        self.ui_client = self.add_ui_client(callback_urls=[
+            'http://localhost:3018/auth/callback'
```

Comment:
It's the tcp port that the UI dev team configured the UI to use, locally.

Reply:
hmm, any way to make this reference an env variable? If someone's running locally and has to use a different port, I'd prefer for this not to break.

Reply:
Sure, we can set it up as an override-able default based on an optional context variable. I'll add that in shortly.

Reply:
FYI - I'll be adding this change to the StaffUsers user pool, not the Board Users one, since we're just working on tearing down the BoardUsers, anyway.